src/engine.py

```
@@ -1576,68 +1611,140 @@ def _get_attended_transfer_helper_settings(self, attended_cfg: Optional[Dict[str
+            campaign_id = str(campaign.get("id") or "").strip() or None
+            if now_utc.tzinfo is None:
+                logger.warning(
+                    "Outbound campaign window rejected: current time is timezone-naive",
+                    campaign_id=campaign_id,
+                )
+                return False
+
             tz_name = str(campaign.get("timezone") or "UTC").strip() or "UTC"
             try:
                 tz = ZoneInfo(tz_name)
             except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)